testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ee/appclient/basic/Status.java

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright The WildFly Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.jboss.as.test.integration.ee.appclient.basic;
+
+/**
+ * Status
+ *
+ * @author Scott Marlow
+ */
+public class Status {
+    String getStatus() {
```

Comment:
@scottmarlow Does this work if this method is public?

I expect it would and I don't expect it to work like this. At runtime this class isn't actually in the same package as AppClientMain even though it looks like it would be from a quick look at the source code.

AppClientMain is in the module for the appclient client subdeployment. This class is in the module for the ear parent deployment. Two different modules == two different classloaders == not the same package despite the same package name.

Reply:
Thanks, making that method public did seem to solve this failure which is good news with regard to ear/lib classes being accessible to the app client container!  I'll close this pr.

Reply:
Actually, we might see if this test is a duplicate of existing logic.  If not, it could be a good test.

Reply:
@scottmarlow Yes, please leave this open. It's good additional coverage.